src/test/java/uk/gov/hmcts/darts/task/runner/impl/AudioLinkingAutomatedTaskTest.java

```diff
@@ -160,13 +157,11 @@ void positiveProcessEvent() {
 
         @Test
         void positiveProcessEventWithBuffer() {
```

Comment:
Seeing as you've amended this test, could you improve the name?

Reply:
Updated